### ISSUE TITLE
Avoid adding translated messages as original messages into the translater_cache table

### ISSUE THREAD

**Issue by sebastianSchmidt86** [humhub/translator-issues#9]
If you translate a message 2 entries will be added to the `translator_cache` table.

Eg:
Original message: this is english

In the database there is the following entry:
```
43aa91130eb9a938054ced61a3f93ea1f579843b9330aec5359666bdb2cc0512 | EN | NULL | 1 | this is english
```

If I translate this message into german, then 2 entries are added:
```
43aa91130eb9a938054ced61a3f93ea1f579843b9330aec5359666bdb2cc0512 | de | NULL | 0 | dies ist Englisch
# The above entry is correct, but the following is superfluous, because this "original" message will never be translated into another language
f883b86066093f08c4e2a11f03edd03aede661255bfe9570b9965db64c338ce0 | DE | NULL | 1 | dies ist Englisch
```

**Comment by yurabakhtin** [comment]
@luke- Fixed in PR https://github.com/humhub/translator/pull/38.

**Comment by luke-** [comment]
Fixed with 1.2.3
